### DISCUSSION TITLE
Release of channel phone.

### DISCUSSION THREAD

**Discussion by protricker** [Sylius/Sylius#12159] [Q&A] [answered]
Hello. The latest release 1.8.6 has no https://github.com/Sylius/Sylius/pull/11950. Could you please provide ETA when new release with channel contact phone number will be available?

**Comment by Zales0123** [comment, marked as answer]
Hello @protricker! Unfortunately, as this is a new feature it will be included in the next minor not patch release (so `1.9.0`). Btw, please, use our [Slack](http://sylius.com/slack) for such questions - we're not using this discussion panel at all 🖖
